tests/sentry/monitors/clock_tasks/test_check_timeout.py

```diff
@@ -20,8 +21,9 @@
 
 
 class MonitorClockTasksCheckTimeoutTest(TestCase):
+    @mock.patch("sentry.monitors.clock_tasks.check_timeout.mark_failed", wraps=mark_failed)
```

Comment:
We actually validate the mark_failed behavior in this test, its' more integrationy